aws/app/lambda/reliability/lib/notifyProcessing.js

```diff
@@ -4,8 +4,8 @@ const { extractFileInputResponses } = require("dataLayer");
 const { retrieveFilesFromReliabilityStorage } = require("s3FileInput");
 
 module.exports = async (submissionID, sendReceipt, formSubmission, language, createdAt) => {
-  const templateID = "92096ac6-1cc5-40ae-9052-fffdb8439a90";
-  const notify = new NotifyClient("https://api.notification.canada.ca", process.env.NOTIFY_API_KEY);
```

Comment:
We can leave the link hardcoded as opposed to adding a variable if its unchanging